docs/platform/api/using-the-linode-cli.md

```diff
@@ -0,0 +1,314 @@
+
+        linode-cli linodes view $linode_id
+
+4.  Boot, shut down, or reboot a Linode:
+
+        linode-cli linodes boot $linode_id
+        linode-cli linodes reboot $linode_id
+        linode-cli linodes shutdown $linode_id
+
+5.  View a list of available IP addresses for a specific Linode:
+
+        linode-cli linodes ips-list $linode_id
+
+6.  Add an additional IP address to a Linode:
+
+    Private IPv4:
+
+        linode-cli linodes ip-add $linode_id --type ipv4 --public false
+
+    Public IPv6:
+
+        linode-cli linodes ip-add $linode_id --type ipv6 --public true
+
+7.  List all disks attached to a Linode:
+
+        linode-cli linodes disks-list $linode_id
+
+Many other actions are available. Use `linode-cli linodes --help` for a complete list.
+
+### Domains
+
+1.  List the domains on your account:
+
+        linode-cli domains list
+
+2.  View all domain records in a specific domain:
+
+        linode-cli domains records-list $domain_id
+
+3.  Delete a domain:
+
+        linode-cli domains delete $domain_id
+
+4.  Create a new A record in a domain:
+
+        linode-cli domains records-create $domain_id --type A --name subdomain --target 192.0.2.0
+
+### NodeBalancers
+
+1.  Create a new NodeBalancer:
+
+        linode-cli nodebalancers create --region us-east --label new-balancer
+
+2.  Create a configuration for a NodeBalancer:
+
+        linode-cli nodebalancers config-create $nodebalancer_id
+
+3.  Attach a Node to a NodeBalancer:
+
+        linode-cli nodebalancers node-create --address 192.0.2.0 --label node-1
+
+4.  To delete a node, you will need the ID of the NodeBalancer, configuration, and node:
+
+        linode-cli nodebalancers node-delete $nodebalancer_id $config_id $node_id
+
+### Block Storage Volumes
+
+1.  List your current Volumes:
+
+        linode-cli volumes list
+
+2.  Create a new Volume, with the size specified in GiB:
+
+        linode-cli volumes create --label my-volume --size 100 --region us-east
+
+    You can also create the Volume and automatically attach it to a specific Linode:
+
+        linode-cli volumes create --label my-volume --size 100  --linode_id $linode_id
+
+3.  Attach or detach the Volume from a Linode:
+
+        linode-cli volumes attach $volume_id --linode_id $linode_id
+        linode-cli volumes detach $volume_id
+
+4.  Resize a Volume (size can only be increased):
+
+        linode-cli volumes resize $volume_id --size 200
+
+5.  Delete a Volume:
+
+        linode-cli volumes delete $volume_id
+
+### Account
+
+You can view or update your account information, add payment methods, view notifications, make payments, create OAuth clients, and more through the `account` action.
+
+1.  View your account:
+
+        linode-cli account view
+
+2.  View all of your OAuth clients:
+
+        linode-cli account clients-list
+
+3.  View your account settings:
+
+        linode-cli account settings
+
+4.  Make a payment:
+
+        linode-cli account payment-create --cvv 123 --usd 20.00
```

Comment:
You should include an example of viewing events and notifications, as those are important and will likely be a common usage of the CLI. 

You might also include some examples of profile management through the CLI, such as viewing/deleting tokens and authorized apps.

Tickets are probably a popular feature to have in here, but they don't display well right now 😕  too much text wrapping.  Might be worth mentioning anyway.